data/top_config.toml

```diff
@@ -1,6 +1,6 @@
 [[blocks]]
 name = "gpio"
-instances = 4 # RPi, Ard, Pmod0, Pmod1
+instances = 5 # RPi, Ard, Pmod0, Pmod1, PmodC
```

Comment:
Should we put pmod c in between the 0 and 1 like was specified in the pin spec?

Reply:
I personally think it's neater to have it appended as an additional, non-standard thing, but no strong feelings either way. Also, what are you referring to when you say pin spec?

Reply:
I'm happy with this current configuration. I think I was referring to the original pinmapping PR, but let's leave this as is for now.